lib/bot.js

```diff
@@ -34,23 +45,25 @@ class Bot {
 
   // --------------------------------------------------------------------------
 
-  init (options: any, cb: CbError): void {
-    if (options && typeof options.verbose !== 'undefined') {
-      this._verbose = options.verbose
-    }
-    getKeybaseUsernameAndDevicename((err, currentDPair) => {
-      if (currentDPair) {
-        this._dPair = currentDPair
-        this._log(`intialized ${currentDPair.username} (device=${currentDPair.devicename})`)
-      }
-      this._initialized = true
-      cb(err)
+  init(options: InitOptions, cb: CbError): void {
+    this._verbose = options.verbose
+
+    keybaseExec({args: ['oneshot', '--username', options.username, '--paperkey', options.paperkey]}, () => {
```

Comment:
This doesn't handle `err` from `keybaseExec`. I was able to send a chat message without providing any options (username or paperkey).